cli/running/running_test.go

```diff
@@ -113,18 +113,32 @@ func Test_CollectInstances(t *testing.T) {
 	instances, err = CollectInstances("script", instancesEnabledPath,
 		integrity.IntegrityCtx{
 			Repository: &mockRepository{},
-		}, true)
+		}, ConfigLoadAll)
 	assert.NoError(t, err)
 	assert.Equal(t, 1, len(instances))
 
 	require.NoError(t, os.Chmod(instancesEnabledPath, 0666))
 	instances, err = CollectInstances("script", instancesEnabledPath,
 		integrity.IntegrityCtx{
 			Repository: &mockRepository{},
-		}, true)
+		}, ConfigLoadAll)
 	assert.ErrorContains(t, err, "script.lua: permission denied")
 	assert.Equal(t, 1, len(instances))
 	require.NoError(t, os.Chmod(instancesEnabledPath, 0755))
+	// Skip instance script searching error.
+	instances, err = CollectInstances("script", instancesEnabledPath,
+		integrity.IntegrityCtx{
+			Repository: &mockRepository{},
+		}, ConfigLoadSkip)
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(instances))
+	// Skip instance script searching error.
+	instances, err = CollectInstances("script", instancesEnabledPath,
+		integrity.IntegrityCtx{
+			Repository: &mockRepository{},
+		}, ConfigLoadCluster)
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(instances))
```

Comment:
Please, add it as a separate tests. It's better to have 5 independent tests that a one big.